projects/igniteui-angular/src/lib/directives/drag-drop/drag-drop.directive.ts

```diff
@@ -918,7 +918,7 @@ export class IgxDragDirective implements AfterContentInit, OnDestroy {
         // Set pointer capture so we detect pointermove even if mouse is out of bounds until ghostElement is created.
         const handleFound = this.dragHandles.find(handle => handle.element.nativeElement === event.target);
         const targetElement = handleFound ? handleFound.element.nativeElement : event.target || this.element.nativeElement;
-        const targetValid = document.body.contains(targetElement);
+        const targetValid = this.isElementInDOM(targetElement);
```

Comment:
If we just want to check if element is in the DOM it might be simpler to use the `isConnected` API. It's supported in all major browsers (https://caniuse.com/mdn-api_node_isconnected) and I think we've already used it in some cases for the `ForOf`.